contracts/smart-contract-wallet/SmartAccount.sol

```diff
@@ -68,11 +68,11 @@ contract SmartAccount is
     
     // Events
     // EOA + Version tracking
-    event ImplementationUpdated(address indexed _scw, string version, address newImplementation);
+    event ImplementationUpdated(address indexed _scw, string indexed version, address indexed newImplementation);
```

Comment:
As per discussion do we need this many indexed filters? @filmakarov suggested this as gas optimisation to have less indexed parameter.

Reply:
hmm I am keeping 3 indexed for every event. post merge we can check all events and optimise what we need to index